packages/file-selector/README.md

```diff
@@ -218,3 +222,59 @@ const MyFileUploadComponent = () => {
 
 export default MyFileUploadComponent;
 ```
+
+##### Custom `helpText`
+
+To provide custom help text, pass it as a child of the `<FileSelector />` component. The help text should be formatted using the `<Typography />` component with the `'caption'` variant.
+
+```tsx
+import React from 'react';
+import { FileSelector } from '@availity/mui-file-selector';
+import { Typography } from '@availity/mui-typography';
+
+const MyComponent = () => {
+  const methods = useForm({
+    defaultValues: {
+      [props.name]: [] as File[],
```

Comment:
Instead of `props.name` we should use the `name` you passed to the component, `"myFiles"`

Reply:
Fixed